### PR TITLE
feat: set up sandbox page

### PR DESCRIPTION
**Pull request by alter-eggo**
> Try out this version of Leather — [download extension builds](https://github.com/leather-wallet/extension/actions/runs/6714550429).

Adds sandbox page with app's components

### CODE REVIEW

**Comment by alter-eggo** [comment]
https://github.com/leather-wallet/extension/assets/46521087/cc3366fb-764e-4e2b-b168-cb06ed9af90f

File: src/app/features/settings-dropdown/settings-dropdown.tsx [hunk 1]
```diff
@@ -166,6 +167,15 @@ export function SettingsDropdown() {
               Lock
             </MenuItem>
           )}
+          {!IS_PROD_ENV && (
```
Comment (pete-watters) [inline src/app/features/settings-dropdown/settings-dropdown.tsx:170]: Maybe we can avoid this and just get people that need it to bookmark the URL? 

Just as I don't think we should mix sandbox stuff with real production code

**Review by pete-watters — APPROVED** [review]

File: src/app/pages/sandbox/sandbox-typography.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { Flex, Stack, styled } from 'leather-styles/jsx';
+
+const textVariants = ['label.01', 'label.02', 'label.03', 'heading.01', 'heading.02', 'heading.03'];
```
Comment (kyranjamie) [inline src/app/pages/sandbox/sandbox-typography.tsx]: Can pull programatically from theme directly?

Comment (alter-eggo) [reply src/app/pages/sandbox/sandbox-typography.tsx]: yes, done

File: src/app/features/settings-dropdown/settings-dropdown.tsx (hunk 1 above)
Comment (alter-eggo) [reply src/app/features/settings-dropdown/settings-dropdown.tsx:170]: it's just for convenience. I believe there is no problem in adding menu item there

**Comment by alter-eggo** [comment]
close in favor of storybook

**Pull request closed without merging**